### PR TITLE
Make it native versioning strategy

### PR DESCRIPTION
**Pull request by Ahmad-Elsayed**

### CODE REVIEW

**Review by NikolaSimsic — APPROVED** [review]

File: content/en/docs/refguide/mobile/distributing-mobile-apps/building-native-apps/how-to-min-versions.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Mendix Studio Pro 10.11 (Latest)
+
+For projects developed with Mendix Studio Pro 10.11, specific builds of **Make It Native 10** are available for Android and iOS. These builds can be accessed via the following URLs:
+
+- **iOS (TestFlight URL)**: [TestFlight URL for Make It Native 10](https://testflight.apple.com/TESTFLIGHT_URL)
+- **Android (Google OpenTesting URL)**: [Google OpenTesting URL for Make It Native 10](https://play.google.com/store/apps/details?id=OPENTESTING_URL)
+
+These builds are updated to support the latest features and improvements introduced in Mendix 10.11 and beyond.
+
```
Comment (UrazAkgultan) [inline content/en/docs/refguide/mobile/distributing-mobile-apps/building-native-apps/how-to-min-versions.md]:
```suggestion
### Mendix Studio Pro 10.11 (Latest)

For projects developed with Mendix Studio Pro 10.11, specific builds of **Make It Native 10** are available for Android and iOS. These builds can be accessed via the following URLs:

- **iOS (TestFlight URL)**: [TestFlight URL for Make It Native 10](https://testflight.apple.com/TESTFLIGHT_URL)
- **Android (Google OpenTesting URL)**: [Google OpenTesting URL for Make It Native 10](https://play.google.com/apps/testing/com.mendix.developerapp.mx10)

These builds are updated to support the latest features and improvements introduced in Mendix 10.11 and beyond.

### How to Join the Public Beta for Our Android App

Follow the steps below to participate in our beta testing program

#### Step-by-Step Guide

1. Join the Google Play Beta Program
   1. Open the Google Play Store on your Android device.
   2. Search for Make it Native 10 in the search bar.
   3. Scroll down to the "Join the beta" section.
   4. Tap on "Join" to become a beta tester.
   5. Confirm your decision to join the beta program.
2. Install the Beta Version
   1. Wait a few minutes after joining the beta program for the beta version to become available.
   2. Go back to the app's Google Play Store page.
   3. Tap on "Update" to install the beta version of the app if it is already installed on your device. If you don’t have the app installed yet, tap on “Install” to get the beta version directly.
3. Leave the Beta Program (Optional)
   1. Open the Google Play Store and navigate to our app's page.
   2. Scroll down to the "Join the beta" section.
   3. Tap on "Leave" to exit the beta program.
   4. Uninstall and reinstall the app if you want to return to the public version.
```

I have drafted a step-by-step guide and updated beta link for users to join the public beta for our Android app. This document will help users understand how to participate in our beta testing program. Please review and consider including it in the current PR.

**Review by UrazAkgultan — APPROVED** [review]

File: content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md [hunk 2]
```diff
@@ -1,22 +1,116 @@
 ---
 title: "Make It Native Apps"
 url: /releasenotes/mobile/make-it-native-parent/
-
+aliases:
```
Comment (ConnorLand) [inline content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md]: @Ahmad-Elsayed are these aliases covering for links pointing to the locations of your old docs which now do not exist? If so they will need to be taken out, and if SP has links pointing to these locations, those links should be changed ASAP.

File: content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md [hunk 3]
```diff
@@ -1,22 +1,119 @@
...
+
+### 2.5 General Support for Make It Native 10
+
+Make It Native 10 will continue to support the latest Mendix Studio Pro MTS versions. This ensures ongoing compatibility and support for the newest updates and features.
+
+### 2.6 Other Mendix Studio Pro versions
+
+For all other versions, [create a Custom Developer App](/refguide/mobile/distributing-mobile-apps/building-native-apps/how-to-devapps/).
+
+## 3 Join MIN Public Beta {#join-min-beta}
```
Comment (ConnorLand) [inline content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md:69]: We need a better description of why someone would want to/must download the beta here.

Comment (ConnorLand) [reply content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md:69]: Also we can bring that reason into the intro.

**Review by ConnorLand — APPROVED** [review]

**Comment by ConnorLand** [comment]
We'll be merging this next week.

File: content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md (hunk 3 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md:69]: I have updated the intro for more information about the need for the beta versions

File: content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md (hunk 2 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/releasenotes/mobile/make-it-native-parent/_index.md]: removed, I will update it in MiN.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**